### ISSUE TITLE
[Regression]: playwright-1.42.0-javadoc.jar contains just the MANIFEST.MF

### ISSUE THREAD

**Issue by m4r71n** [microsoft/playwright-java#1533]
### Last Good Version

1.41.2

### First Bad Version

1.42.0

### Steps to reproduce

Please see the available `playwright-1.42.0-javadoc.jar` on maven central repository:
https://repo1.maven.org/maven2/com/microsoft/playwright/playwright/1.42.0/playwright-1.42.0-javadoc.jar

### Expected behavior

Available Javadoc

### Actual behavior

Missing Javadoc

### Environment

Maven

**Comment by yury-s** [comment]
This regressed in https://github.com/microsoft/playwright-java/pull/1484

**Comment by yury-s** [comment]
@mxschmitt fyi

**Comment by yury-s** [comment]
This is a regression in `maven-javadoc-plugin` introduced somewhere between 3.2.0 and 3.4.0 or 3.3.0 for that matter.

**Comment by yury-s** [comment]
Debugging notes. Javadoc options generated by the plugin have changed to include `-source 8`:

```
$ diff good.txt bad.txt 
5a6,7
> -source
> '8'
24a27,28
> 'https://docs.oracle.com/javase/8/docs/api' '/Users/yurys/playwright-java/playwright/target/javadoc-bundle-options'
> -linkoffline
```

The options are from the following:
```
Command line was: /Library/Java/JavaVirtualMachines/jdk-21.jdk/Contents/Home/bin/javadoc @options

Refer to the generated Javadoc files in '/Users/yurys/playwright-java/playwright/target/site/apidocs' dir.
```

**Comment by m4r71n** [comment]
Fixed with 1.43.0. Thanks!
